dask/dataframe/core.py

```diff
@@ -1026,6 +1026,10 @@ def elemwise(op, *args, **kwargs):
 
     if other:
         op2 = partial_by_order(op, other)
+        other_argname = [o[1] for o in other]
+        if len(other_argname) <= 1:
+            other_argname = other_argname[0]
+        op2.__name__ = '{0}({1})'.format(op.__name__, other_argname)
```

Comment:
Actually, spoke too soon. This case is actually fine, as the deterministic name is chosen based on other factors. I agree with Matt though that this should be pushed into `partial_by_order`.